apps/app-router/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "app-router",
```

```suggestion
  "name": "nextjs-app-router",
```

We can provide examples with other adapters, not just Next.js, so it's probably a good idea to add a prefix to the package and folder names.

---

I missed that part 🔥, thank you! I think it would be a good idea to change the folder name as well.

I'll work on it